src/keeper.ts

```diff
@@ -264,6 +292,88 @@ class Keeper {
         component: "Indexer",
       });
     });
+
+    // update volume metrics
+    await this.updateVolumeMetrics(deps);
+
+    // update open interest metrics
+    await this.updateOIMetrics(deps);
+  }
+
+  pushTradeToVolumeQueue(size: BigNumber, lastPrice: BigNumber) {
+    const tradeSizeUSD = wei(size)
+      .abs()
+      .mul(lastPrice)
+      .div(UNIT)
```

Comment:
We do we divide by UNIT. Does that help with overflow/underflow?

Reply:
because both numbers are decimal (price and size), so we need to divide by `1e18` to get back to 18 decimal precision

Reply:
Ah yep. I think thats needed when using BigNumber. But the wei lib takes care of that.
https://codesandbox.io/s/divine-waterfall-bpwzps?file=/src/index.ts
Happy to leave it in though